sdk/identity/identity/samples/v2/javascript/clientSecretCredential.js

```diff
@@ -1,6 +1,10 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/**
+ * @summary Authenticates with a client and a client's secret.
```

Comment:
The word client is overloaded in this context and could be confusing. Should we use "application" instead?

Reply:
"App Registration"?

Reply:
can do!